config/watcher.go

```diff
@@ -139,6 +139,7 @@ func newFSWatcher(ctx context.Context, configPath string, logger logging.Logger)
 			case event := <-fsWatcher.Events:
 				if event.Op&fsnotify.Write == fsnotify.Write {
 					debounced(func() {
+						logger.Info("On-disk config file changed. Reconfiguring.")
```

Comment:
Haha agree is very helpful 🙏🏻 . Can you change "Reconfiguring" to "Reading changes" or something? I don't think we'll necessarily `Reconfigure` anything unless there's a salient change.

Reply:
LMK